service/raft/raft_group0.cc

```diff
@@ -87,6 +87,10 @@ namespace service {
 static logging::logger group0_log("raft_group0");
 static logging::logger upgrade_log("raft_group0_upgrade");
 
+// TODO: change the links from master to stable/5.2 after 5.2 is released
+static const auto raft_upgrade_doc = "https://docs.scylladb.com/master/architecture/raft.html#verifying-that-the-internal-raft-upgrade-procedure-finished-successfully";
```

Comment:
Shouldn't we change it to 5.2 before 5.2 is released, so that the 5.2 binary refers to the "5.2" path? Should this depend on a variable set by the build system? It will become stale if it requires a manual process.

Reply:
You mean before we branch? I was planning to change it after we branch, on the 5.2 branch.
But maybe it makes sense to change it right now... Although currently the link works, if I change it to refer to non-existing (yet) path it will stop working.

Also, I don't actually know what the path will be :D right now we only have `stable` and `master`, where `stable` corresponds to 5.1, but once we release 5.2, I don't know what it will be. cc @annastuchlik 

Making it depend on a build system variable would be an interesting solution... Although we don't have the necessary scaffolding to do stuff like this right now.

Reply:
We shouldn't change it to 5.2 before branching, of course. We should change it on 5.2 before releasing it. 

But I think we also should have an automated process which makes sure one you commit something to master and we branch, say 5.2, the code built from 5.2 will refer to the documentation for 5.2.

Reply:
https://docs.scylladb.com/stable/upgrade/ is a fairly stable link, but I wouldn't even use that. "Scylla release upgrade procedure" is easy to google and is more likely to stay relevant over time.

Reply:
Except "Scylla release upgrade procedure" is not what the link is pointing to, you could google "scylla internal raft upgrade", you can find https://docs.scylladb.com/stable/architecture/raft.html this way but it's not the first link and it's not pointing to the exact section that I wanted to point to here.

Reply:
@kbr-scylla `stable `is and should always be the latest released stable version. As soon as we release 5.2, I'm going to change the configuration so that `stable` (default) is 5.2.

Reply:
We'll change this link to use `stable` after branching then.